Sources/PrimerSDK/Classes/User Interface/Root/PrimerCardFormViewController.swift

```diff
@@ -78,15 +104,35 @@ class PrimerCardFormViewController: PrimerFormViewController {
             verticalStackView.addArrangedSubview(saveCardSwitchContainerStackView)
             saveCardSwitchContainerStackView.isHidden = true
         }
+    }
+    
+    private func configureZipCodeFieldRow() {
+        zipCodeFieldRow.addArrangedSubview(formPaymentMethodTokenizationViewModel.zipCodeContainerView)
+        zipCodeFieldRow.addArrangedSubview(UIView())
```

Comment:
Prefer `PrimerView`. Also do it for other instances of `UIView`, I probably didn't have `PrimerView` when I wrote this class.